src/common/perf_counters.cc

```diff
@@ -371,6 +414,10 @@ void PerfCounters::dump_formatted_generic(Formatter *f, bool schema,
       continue;
     }
 
+    if (!d->accessed && dump_labeled) {
```

Comment:
If we can always filter independent of labels I'd be all for it, but I'm not sure how it would interact with existing use cases for perf counters or tooling. In the comment I just wrote I asked a few people if they know how `perf dumps` are used currently.